packages/composites/outlier-detection/src/adapter.ts

```diff
@@ -27,8 +30,8 @@ const customParams = {
   onchain_threshold: false,
 }
 
-const execute: ExecuteWithConfig<Config> = async (input, _, config) => {
-  const paramOptions = makeOptions(config)
+const execute: ExecuteWithConfig<Config> = async (input, _, __) => {
+  const paramOptions = makeOptions(input.data.source.split(','), input.data.check.split(','))
```

Comment:
This doesn't seem very useful to use the input parameters themself to validate what the input parameters are, because it will always come back as valid.

This is the area where using `@chainlink/ea` could be useful (such as how composite `reference transform` does)

Reply:
Skipping this for now as including the `lego` transform threw an error when starting.  This seems to happen in the `reference transform` adapter as well.  We can investigate this again when you are back.

```
/Users/christophersastropranoto/Documents/external-adapters-js/.pnp.cjs:36131
    throw firstError;
    ^

Error: @renproject/rpc tried to access bn.js, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: bn.js
Required by: @renproject/rpc@npm:2.5.1 (via /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/pack/)

Require stack:
- /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/pack/marshal.js
- /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/transaction.js
- /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/renVMProvider.js
- /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/index.js
- /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/index.js
- /Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-ren-npm-2.5.1-afd5050000-f736004eba.zip/node_modules/@renproject/ren/build/main/index.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/sources/renvm-address-set/dist/adapter.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/sources/renvm-address-set/dist/index.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/core/legos/dist/sources.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/core/legos/dist/index.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/composites/outlier-detection/dist/config.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/composites/outlier-detection/dist/index.js
- /Users/christophersastropranoto/Documents/external-adapters-js/packages/composites/outlier-detection/[eval]
    at Function.external_module_.Module._resolveFilename (/Users/christophersastropranoto/Documents/external-adapters-js/.pnp.cjs:36130:55)
    at Function.external_module_.Module._load (/Users/christophersastropranoto/Documents/external-adapters-js/.pnp.cjs:35929:48)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/pack/marshal.js:8:33)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.external_module_.Module._load (/Users/christophersastropranoto/Documents/external-adapters-js/.pnp.cjs:35979:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/Users/christophersastropranoto/Documents/external-adapters-js/.yarn/cache/@renproject-rpc-npm-2.5.1-f230905ff6-3678ce20db.zip/node_modules/@renproject/rpc/build/main/v2/transaction.js:5:19)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.external_module_.Module._load (/Users/christophersastropranoto/Documents/external-adapters-js/.pnp.cjs:35979:14)
```